### ISSUE TITLE
Table: Some rows are hidden when row.selected is set to `true` on table load

### ISSUE THREAD

**Issue by kherP** [sebgroup/react-components#266]
**Describe the bug**
When some rows of the data is set with `selected: true`, the rows with `selected: false` are hidden on table load and the rows are now not able to uncheck.

**To Reproduce**
1. Define data with some rows with `selected: true` and some with `selected: false`
2. Load data with table component
3. Notice that some rows are missing [NOK]
4. Click on one of the checkboxes
5. The checkbox is not able to uncheck [NOK]

**Expected behavior**
Table should allow to have pre-selected rows and show all rows and allow to uncheck pre-selected rows.

**Screenshots**
```
(5) [{…}, {…}, {…}, {…}, {…}]
0:
id: "quiet-1"
firstName: "toothpaste"
lastName: "visitor"
age: 27
visits: 96
progress: 24
status: "relationship"
subRows: (5) [{…}, {…}, {…}, {…}, {…}]
rowContentDetail: {$$typeof: Symbol(react.element), type: "p", key: null, ref: null, props: {…}, …}
selected: true
__proto__: Object
1:
id: "twist-9"
firstName: "donkey"
lastName: "pump"
age: 6
visits: 38
progress: 70
status: "single"
subRows: (5) [{…}, {…}, {…}, {…}, {…}]
rowContentDetail: {$$typeof: Symbol(react.element), type: "p", key: null, ref: null, props: {…}, …}
selected: false
__proto__: Object
2:
id: "burn-9"
firstName: "riddle"
lastName: "fifth"
age: 0
visits: 21
progress: 38
status: "single"
subRows: (5) [{…}, {…}, {…}, {…}, {…}]
rowContentDetail: {$$typeof: Symbol(react.element), type: "p", key: null, ref: null, props: {…}, …}
selected: true
__proto__: Object
3:
id: "act-9"
firstName: "harbor"
lastName: "plants"
age: 25
visits: 94
progress: 86
status: "single"
subRows: (5) [{…}, {…}, {…}, {…}, {…}]
rowContentDetail: {$$typeof: Symbol(react.element), type: "p", key: null, ref: null, props: {…}, …}
selected: false
__proto__: Object
4:
id: "railway-6"
firstName: "trail"
lastName: "cub"
age: 26
visits: 80
progress: 89
status: "single"
subRows: (5) [{…}, {…}, {…}, {…}, {…}]
rowContentDetail: {$$typeof: Symbol(react.element), type: "p", key: null, ref: null, props: {…}, …}
selected: true
__proto__: Object
length: 5
__proto__: Array(0)
```
![image](https://user-images.githubusercontent.com/12553063/79036244-f625ce00-7bf8-11ea-8287-0d81ec720f92.png)

**Additional context**

**Comment by DanSallau** [comment]
What is NOK ?

**Comment by DanSallau** [comment]
fixed by #254
